Code/Tools/AssetProcessor/native/ui/MainWindow.cpp

```diff
@@ -1557,7 +1557,13 @@ void MainWindow::OnAssetProcessorStatusChanged(const AssetProcessor::AssetProces
 
         if (m_processJobsCount + m_createJobCount > 0)
         {
-            text = tr("Working, analyzing jobs remaining %1, processing jobs remaining %2...").arg(m_createJobCount).arg(m_processJobsCount);
+            text += tr("Working, analyzing jobs remaining %1, processing jobs remaining %2...").arg(m_createJobCount).arg(m_processJobsCount);
+
+            if (!entry.m_extraInfo.isEmpty())
+            {
+                text += tr("<p style='font-size:small;'>%1</p>").arg(entry.m_extraInfo);
```

Comment:
would this string ever be internationalized? you can probably just use a QString.

Reply:
I have no problem keeping tr() in there for future reference.  its my understanding that tr() passes it thru the localization system, and returns a QString anyway, so the only difference between using tr() and QString in this case would be the loop thru the translations system.  tr() also converts it to a QString on entry, and QStrings are refcounted copy-on-write objects that compute their hash on creation (AFAIK?  Might be lazily) so the cost to call this is the cost of an extra dereference and a hash-table lookup in the translation table.